rockcraft/services/lifecycle.py

```diff
@@ -119,4 +119,32 @@ def _post_prime_callback(step_info: StepInfo) -> bool:
     files = step_info.state.files if step_info.state else set()
 
     layers.prune_prime_files(prime_dir, files, base_layer_dir)
+
+    _python_usrmerge_fix(step_info)
+
     return True
+
+
+def _python_usrmerge_fix(step_info: StepInfo):
+    """Fix 'lib64' symlinks created by the Python plugin on ubuntu@24.04 projects."""
+    if step_info.project_info.base != "ubuntu@24.04":
```

Comment:
I'm guessing this is going to be a problem with future bases too?

Reply:
very likely, I guess I'm trying to be extra conservative by making the fix as localized as possible. And I kind of want to see this break for "ubuntu@26.04" too so we can come and update this